src/tests/end-to-end/common/browser.ts

```diff
@@ -189,6 +190,12 @@ export class Browser {
 
     private async getActivePageTabId(): Promise<number> {
         const backgroundPage = await this.background();
+
+        // Check chrome.tabs is initialized
+        while (await backgroundPage.evaluate(() => chrome.tabs == null, null)) {
```

Comment:
Can this loop have a max timeout in case something goes wrong and chrome.tabs is never initialized?